worker/index.ts

```diff
@@ -50,7 +50,7 @@ const { worker, RoomDO, AuthDO } = createReflectServer({
     await clearCursorAndSelectionState(write, { id: write.clientID });
   },
   getLogSinks,
-  getLogLevel: () => "info",
+  getLogLevel: () => "debug",
```

Comment:
Just to confirm, this doesn't negatively affect perf? I think I recall that monday perhaps doesn't enable debug logging for perf reasons? @aboodman would know.

Reply:
Monday definitely complained about this. I'm not sure about replidraw.

Reply:
I think it does, I just have it turned up to debug while i am testing that the dd31 changes work.